technology/deployments.md

```diff
@@ -38,7 +38,7 @@
 | Delegate | [`2.6.9`](https://www.npmjs.com/package/@airswap/delegate/v/2.6.9) | [`0xe01cE51C3CB0B68b5Fa07CCBD9544AeEcd006e6E`](https://rinkeby.etherscan.io/address/0xe01cE51C3CB0B68b5Fa07CCBD9544AeEcd006e6E) | [`953956f`](https://github.com/airswap/airswap-protocols/tree/953956f308c65ec53d1f1b20d35f47fe04b936af) |
 | Staking | [`2.3.0`](https://www.npmjs.com/package/@airswap/staking/v/2.3.0) | [`0x7c04c63b971D23f39db2D0eaecB6AE5537Bcd467`](https://rinkeby.etherscan.io/address/0x7c04c63b971D23f39db2D0eaecB6AE5537Bcd467) | [`6349b6f`](https://github.com/airswap/airswap-protocols/tree/6349b6f0d9aad4e04202a4981697a4f87bb12b3a) |
 | Pool | [`1.1.0`](https://www.npmjs.com/package/@airswap/pool/v/1.1.0) | [`0xdeD442725364710f5501a85Eb9Ff49CC45851693`](https://rinkeby.etherscan.io/address/0xdeD442725364710f5501a85Eb9Ff49CC45851693) | [`eef21f7`](https://github.com/airswap/airswap-protocols/tree/eef21f7b713983a471de0a3dae811cbb885941bb) |
-| Wrapper | [`3.6.9`](https://www.npmjs.com/package/@airswap/wrapper/v/3.6.9) | [`0x8C80e2c9C5244C2283Da85396dde6b7af4ebaA31`](https://rinkeby.etherscan.io/address/0x8C80e2c9C5244C2283Da85396dde6b7af4ebaA31) | [`953956f`](https://github.com/airswap/airswap-protocols/tree/953956f308c65ec53d1f1b20d35f47fe04b936af) |
+| Wrapper | [`2.5.2`](https://www.npmjs.com/package/@airswap/wrapper/v/2.5.2) | [`0xc778417E063141139Fce010982780140Aa0cD5Ab`](https://rinkeby.etherscan.io/address/0xc778417E063141139Fce010982780140Aa0cD5Ab) | [`953956f`](https://github.com/airswap/airswap-protocols/tree/953956f308c65ec53d1f1b20d35f47fe04b936af) |
```

Comment:
This is for Rinkeby WETH. Wrapper is `0xC176Ee8eFB929c918B68914A73Ac3329AfAdD386`

See: https://github.com/airswap/airswap-protocols/blob/main/source/wrapper/deploys.js

Reply:
oh okay, never mind then  ;)